experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -98,19 +98,31 @@ def get(self, *args, **kwargs):
 
 
 def build_experiment_context(experiment):
-    doc_base_url = "https://mozilla.github.io/metric-hub/outcomes/"
+    outcome_doc_base_url = "https://mozilla.github.io/metric-hub/outcomes/"
     primary_outcome_links = [
-        (outcome, f"{doc_base_url}{experiment.application}/{outcome}")
+        (outcome, f"{outcome_doc_base_url}{experiment.application}/{outcome}")
         for outcome in experiment.primary_outcomes
     ]
     secondary_outcome_links = [
-        (outcome, f"{doc_base_url}{experiment.application}/{outcome}")
+        (outcome, f"{outcome_doc_base_url}{experiment.application}/{outcome}")
         for outcome in experiment.secondary_outcomes
     ]
+
+    segment_doc_base_url = "https://mozilla.github.io/metric-hub/segments/"
+    segment_links = [
+        (
+            segment,
+            f"{segment_doc_base_url}"
+            f"{experiment.application.replace('-', '_')}/"
+            f"#{segment}",
```

Comment:
Was this too long to fit on one line? I was confused at first about why we had a 4-tuple here (because I didn't notice the missing commas). What do you think about using `\` and `+` at the end/beginning of each line so it's clearer that it's a single string?